src/javascripts/components/cookie-functions.js

```diff
@@ -237,7 +244,7 @@ function setCookie (name, value, options) {
     if (typeof options === 'undefined') {
       options = {}
     }
-    var cookieString = name + '=' + value + '; path=/'
+    var cookieString = name + '=' + value + ';path=/'
```

Comment:
Why is this change needed?

Reply:
Oops, this wasn't intentional - will remove.